.travis.yml

```diff
@@ -26,11 +26,11 @@ matrix:
         - DOMAIN_TEST=chatbot-dev.brave.coop
```

Comment:
Also, please remember a CHANGELOG entry and your test plan! Thanks!

Reply:
Ah yes! Will do!

Reply:
Done!